utils/ifuse/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ifuse
+PKG_VERSION:=1.1.5
```

Comment:
Does PKG_VERSION make sense here?

Reply:
For those packages that have no release and were switched to the latest commit, `PKG_VERSION` defines the string returned by `--version`, for example:

``` bash
ifuse --version

# PKG_VERSION:=1.1.5
ifuse 1.1.5

# PKG_VERSION not set in Makefile, seems to be auto-generated
ifuse 2023.01.06~814a0e38
```

The [commit](https://github.com/libimobiledevice/ifuse/commit/38c9f456fb4fc007ea99dff70eda32950a39fd2a) right after the last release says *Post release version bump to 1.1.5*. So if we honour to display that version, we need to set it in `PKG_VERSION`. If we prefer the auto-generated version, we should not set `PKG_VERSION`.

Note that `PKG_VERSION` is later used for `PACKAGE_VERSION` which is required for the build when we use `autoreconf`.
``` Makefile
CONFIGURE_ARGS += \
	PACKAGE_VERSION=$(PKG_VERSION)
```

Let me know if I should keep it or remove it.